draft-rha-jose-hpke-encrypt.md

```diff
@@ -306,6 +341,47 @@ In the above example, the JWE Protected Header value is:
 }
 ~~~
 
+~~~
+{
+  "protected": "eyJlbmMiOiJBMTI4R0NNIiwiZXBrIjp7Imt0eSI6IkVLIiwiZWsiOiJCUFJUS244bVFMNGhOMWF5b2tSOGdrcFR5NUhRbGQ0TjBIWFhCOWNYdGpVSVEzN3pzSkRMN1R1Z1ZrbUQxYUZZVHgtMGJNMHRmeHplakxjdFNES2pNUXMifX0",
+  "encrypted_key": "zR0ArfrVVRQ9-X_heDU2riwx36QxLBffRrKAWU-tLC4",
+  "iv": "o3v11Hw6gUxUN-pY",
+  "ciphertext": "Ny-2IDGHMI3MzVsUAVMGNoKAZfoewTZ1dkAIBikPy4eZUfHW_LPhhKpD6Mf4zYGkhAeLwGgJKjyDoFIj0EuDsEtJ",
+  "tag": "0sfzHJvxVoWt02EPxMTh8w"
+}
```

Comment:
Likewise, this example is missing an "alg" value.

Reply:
Good catch, fixed.